apps/api/src/domain/models/shared/access-control/access-control-list.entity.ts

```diff
@@ -65,4 +67,19 @@ export class AccessControlList extends BaseDomainModel {
     canGroup(groupId: AggregateId) {
         return this.allowedGroupIds.includes(groupId);
     }
+
+    // TODO Should `isPublished` be part of the ACL?!
```

Comment:
That's a really interesting point. This might look like:
```ts
class AccessControlList{
   boolean isPublished;
    // ...
   boolean canUserWithGroups(u){
    if(this.isPublished){return true;}
}
}
```
Let's hold of on this until we have finished event sourcing all resources. At that point, we can change `handleResourcePublished` in the domain and the event consumer in the query layer to write to an ACL instead.